schema/objects/patch-info.schema.yaml

```
@@ -0,0 +1,14 @@
+title: Patch Info
+description:
+  Defines ephemeral patch information related to the Cloud collaborative editing
+  feature. This information will only be found behind-the-scenes in Cloud
+  documents and won't be relevant or visible to users parsing or generating
+  their own Sketch documents.
+type: object
+properties:
+  _class: { const: MSImmutablePatchInfo }
+  status: { $ref: ./co-edit-status-info.schema.yaml }
+  lastIntegratedPatchID: { $ref: ../utils/uuid.schema.yaml }
+  localPatches:
+    type: array
+    items: { $ref: ./file-ref.schema.yaml }
```

```suggestion
    items: { $ref: ./file-ref.schema.yaml }
  receivedPatches:
     type: array
     items: { $ref: ./file-ref.schema.yaml }
```
The code also has a path which will write `receivedPatches`. In reality there should never be a scenario in which these get written, but since the code supports it we should probably include them here. The content would have exactly the same structure as that of `localPatches`